using-mobile-extensions/mobile-core/identity/README.md

```diff
@@ -132,7 +132,7 @@ initSDK() {
 {% endtabs %}
 
 {% hint style="info" %}
-Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each visitor in the Adobe Experience Cloud and is similar to the following value:`65002602839283012066007008864929833047`.
+Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each visitor in the Adobe Experience Cloud and is a 32 character length ID.
```

Comment:
- Please replace "or ECID" with (ECID)
- Please change "32 character" to "32-character"
- Can we remove "length" and just say "32-character ID"?